scripts/sexdescriptions.gd

```diff
@@ -0,0 +1,111 @@
+var C_FULL_BODY_SCALES = 'fullscales'
```

Comment:
This mod already patches this file. Any changes should be performed there.

Reply:
Fixed